Source/WebCore/html/HTMLInputElement.cpp

```diff
@@ -1886,6 +1886,25 @@ bool HTMLInputElement::isEmptyValue() const
     return m_inputType->isEmptyValue();
 }
 
+bool HTMLInputElement::isDevolvable() const
+{
+    return m_inputType->isTextType()
```

Comment:
I think you can just return `true` here.

From https://html.spec.whatwg.org/multipage/rendering.html#widgets:
```
The following elements can have a native appearance for the purpose of the CSS appearance property.

- button
- input
- meter
- progress
- select
- textarea
```

Reply:
It's a little bit more complicated. For example, `<input>` whose `type` is checkbox or radio group is non-devolvable:
https://html.spec.whatwg.org/multipage/rendering.html#the-input-element-as-a-checkbox-and-radio-button-widgets